test/integration/uds_integration_test.cc

```diff
@@ -84,6 +93,24 @@ HttpIntegrationTest::ConnectionCreationFunction UdsListenerIntegrationTest::crea
   };
 }
 
+TEST_P(UdsListenerIntegrationTest, TestSocketMode) {
+  if (abstract_namespace_) {
+    // stat(2) against sockets in abstract namespace is not possible
+    GTEST_SKIP();
+  }
+
+  initialize();
+  struct stat listener_stat;
+  EXPECT_EQ(::stat(getListenerSocketName().c_str(), &listener_stat), 0);
```

Comment:
use `OsSysCallsSingleton` for `stat` syscall?

Reply:
overlooked that, fixed https://github.com/envoyproxy/envoy/pull/13493/commits/16aa22fbdef81a8a25ae1e724853a9537c3d6427